.travis.yml

```diff
@@ -4,9 +4,12 @@ sudo: false
 before_script:
   - export MAVEN_SKIP_RC=true
 
+before_cache:
+  - rm -rf $HOME/.m2/repository/org/locationtech/geomesa
 cache:
   directories:
   - $HOME/.m2
+  - $HOME/zinc
```

Comment:
i think the reason we weren't using zinc is b/c we didn't have enough memory to run it